tests/path.rs

```diff
@@ -0,0 +1,69 @@
+#![cfg_attr(
+    feature = "dev",
+    allow(dead_code, unused_variables, unused_imports, unreachable_code)
+)]
+#![deny(clippy::all)]
+
+#[path = "./utils.rs"]
+mod utils;
+
+use check_protocols::R;
+use test_utils::trim_margin;
+use utils::test_run;
+
+#[test]
+fn looks_up_step_executable_in_path() -> R<()> {
+    test_run(
+        r##"
+            |#!/usr/bin/env bash
+            |cp
+        "##,
+        r##"
+            |protocol:
+            |  - cp
+        "##,
+        Ok(()),
+    )?;
+    Ok(())
+}
+
+#[test]
+fn looks_up_unmocked_command_executable_in_path() -> R<()> {
+    test_run(
+        r##"
+            |#!/usr/bin/env bash
+            |ls $(dirname dir/file)
```

Comment:
This line can be simplified to just `dirname dir/file`. (In general I'm trying to keep the tests as simple as possible and only test one thing at a time.)

Reply:
Sounds good. I simplified this test to `ls > /dev/null` to get a single command that will not crash the script, and that will not produce output.